gstd/gstd.c

```diff
@@ -100,6 +101,10 @@ main (gint argc, gchar * argv[])
     {"gst-log-filename", 'd', 0, G_OPTION_ARG_FILENAME, &gstlogfile,
         "Create gst.log file to path", NULL}
     ,
+    {"no-log", 'L', 0, G_OPTION_ARG_NONE, &nolog,
+          "Disable gstd and gstd file logging while in daemon mode (Takes precedence over -l and -d)",
```

Comment:
The description sounds weird. Perhaps something like:

```
Disable gstd console and file logging while in daemon mode
```

Reply:
The text "Disable gstd and gstd file logging while in daemon mode (Takes precedence over -l and -d)" doesn't make sense.  Maybe you mean "Disable file logging when gstd is running in foreground or daemon mode.  Takes precedence over -l and -d."